<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-MSPM6D0" failures="1" tests="5" name="Test" time="32.258" errors="0" timestamp="2025-02-02T15:52:24 IST">
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="1.831"/>
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="10.939"/>
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="1.485">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC003_LoginDDT.verify_loginDDT(TC003_LoginDDT.java:56)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="11.003"/>
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="1.656"/>
</testsuite> <!-- Test -->
